packages/cli/src/rpc/explain/navie/navie-local.ts

```diff
@@ -40,8 +40,21 @@ class LocalHistory {
   }
 }
 
+const OPTION_SETTERS: Record<string, (explainOptions: Explain.ExplainOptions, value: string | number) => void> = {
+  tokenLimit: (explainOptions, value) => {
+    explainOptions.tokenLimit = value as number;
+  },
+  temperature: (explainOptions, value) => {
+    explainOptions.temperature = value as number;
+  },
```

Comment:
You'd need to use something like `parseInt` to effectively coerce `value` into a string. This will only quiet the type checker, the underlying value will still be a string.